__tests__/runtime.test.js

```diff
@@ -137,4 +137,9 @@ describe('browser.runtime', () => {
     browser.runtime.openOptionsPage();
     expect(browser.runtime.openOptionsPage).toHaveBeenCalledTimes(1);
   });
+  test('getManifest', () => {
+    expect(jest.isMockFunction(browser.runtime.getManifest)).toBe(true);
+    browser.runtime.getManifest();
```

Comment:
If this is always going to return this manifest version, can you test for that as well in the first call?

```suggestion
    expect(browser.runtime.getManifest()).toEqual({ manifest_version: 3 });
```